README.md

```diff
@@ -53,7 +53,8 @@ More information about configuring Logstash can be found in the [logstash config
 | --- | --- | --- |
 | **path** | The plugin writes events to temporary files before sending them to ADX. This parameter includes a path where files should be written and a time expression for file rotation to trigger an upload to the ADX service. The example above shows how to rotate the files every minute and check the Logstash docs for more information on time expressions. | Required
 | **ingest_url** | The Kusto endpoint for ingestion-related communication. See it on the Azure Portal.| Required|
-| **app_id, app_key, app_tenant**| Credentials required to connect to the ADX service. Be sure to use an application with 'ingest' privileges. | Required|
+| **app_id, app_key, app_tenant**| Credentials required to connect to the ADX service. Be sure to use an application with 'ingest' privileges. | Optional|
+| **managed_identity**| Managed identity id (ClientId GUID in case of User managed identity or system in case of system managed identity). The id needs to have 'ingest' privileges | Optional|
```

Comment:
I think it's better to have two fields - a boolean system managed identity, and a string/guid user managed identity.
The magic string "system" isn't very clear, and I don't think we use it elsewhere (correct me if I'm wrong).

If there are multiple methods enabled, return an error.

Reply:
You are right that it is not very intiutive and is a magic string. There are a few connectors we use this convention (NLOG, Serilog).

The inspiration came from this : 

https://learn.microsoft.com/en-us/azure/data-explorer/kusto/api/connection-strings/kusto

ManagedServiceIdentity | EmbeddedManagedIdentity | A string value that instructs the client which application identity to use with managed identity authentication. Use system to indicate the system-assigned identity.
-- | -- | --

We can remove it and make an additional parameter too, we followed this because of this convention that was present (we can remove it too ). What do you think..

Reply:
Updated this readme as well